### ISSUE TITLE
add ability to switch between services for a meta contact

### ISSUE THREAD

**Issue by tonyguadagno** [miranda-ng/miranda-ng#1629]
**Is your feature request related to a problem? Please describe.**
For a meta contact, sometimes i want the message to go over a particular service.  if both services are available, i cannot control which is used for the chat.

**Describe the solution you'd like**
in the chat window, where you list the service being used, there should be a way (dropdown or hotlink) to choose the service this chat will use

thanks, nice client!

**Comment by dartraiden** [comment]
Right-click on meta in contact list → Subcontacts → Click on the desired subcontact → Message
